zio-http/src/test/scala/zio/http/middleware/WebSpec.scala

```diff
@@ -47,6 +47,22 @@ object WebSpec extends ZIOSpecDefault with HttpAppTestExtensions { self =>
         assertZIO(program)(equalTo(Vector("404 GET /health 0ms\n")))
       },
     ),
+    suite("logging")(
+      test("log status method url and time") {
+        val program = runApp(app @@ logResponseDefault) *> ZTestLogger.logOutput
+        for {
+          res <- program
+          logMessages = res.map(_.message())
+        } yield assertTrue(logMessages == Chunk("200 GET /health 1000ms"))
+      },
+      test("log 404 status method url and time") {
+        val program = runApp(Http.empty ++ Http.notFound @@ logResponseDefault) *> ZTestLogger.logOutput
```

Comment:
```suggestion
        val program = runApp(Http.empty @@ logResponseDefault) *> ZTestLogger.logOutput
```
We should have to compose it with a `notFound`. Logger should work automatically for not-found statuses.